hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/sender/http/JettyMessageSender.java

```diff
@@ -25,6 +27,14 @@
     private final long timeout;
     private final MetadataAppender<Request> metadataAppender;
 
+    private final Function<ContentType, String> contentTypeToMediaType = contentType -> {
+        if (ContentType.AVRO.equals(contentType)) {
+            return pl.allegro.tech.hermes.consumers.consumer.sender.http.MediaType.AVRO;
```

Comment:
it would be nice to statically import this constant or change name to avoid conflicts (APPLICATION_AVRO?)... then it would be possible to write simple elvis one-liner here...